src/models/stores/serial.ts

```diff
@@ -1,4 +1,5 @@
 import { NodeChannelInfo } from "src/plugins/dataflow/model/utilities/channel";
+import { NodeLiveOutputTypes } from "../../plugins/dataflow/model/utilities/node";
```

Comment:
One more thing we should watch out for is this dependency. This might cause the main CLUE javascript bundle to start including all of the dataflow code and its dependencies. This can be checked by running `npm build` and then comparing the sizes of the javascript files with those recorded in the last release that recorded this info: 
https://github.com/concord-consortium/collaborative-learning/releases/tag/clue-3.8.0

Just for reference, @433eros this is why it is useful to record the file sizes. There are other ways to see what is included in each javascript bundle but this course grained size based approach can be done without extra tools.

Reply:
As per our slack discussion - it does look like some dependencies were moved "up."  What I don't totally understand is why the increase is only by 1K while the decrease is 16K?

**v3.9.0**
index `45K`
Dataflow `751K` 

**This PR**
index `46K`
Dataflow `735K`